inventory_test.go

```diff
@@ -1,78 +1,114 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
 	"os"
 	"testing"
 )
 
 const (
-	rackhdUrl  = "http://127.0.0.1:5000"
 	configPath = "./config.test.yaml"
 )
 
-func TestConfigReads(t *testing.T) {
-	var props props
+func setEnvironmentVars(rackhdUrl string, configPath string) error {
 	err := os.Setenv(RackHdApiUrlEnvVarName, rackhdUrl)
 	if err != nil {
-		t.Errorf("%s\n", err)
+
+		return err
 	}
 	err = os.Setenv(AnsibleRackHdConfigPath, configPath)
```

Comment:
I think this is the only var that makes sense to have has an env variable. Still we shouldn't be using `os`, we can do everything with `viper`